test/server/graphql/v2/mutation/ExpenseMutations.test.js

```diff
@@ -1086,6 +1086,19 @@ describe('server/graphql/v2/mutation/ExpenseMutations', () => {
             `Payment being processed: January Invoice for ${collective.name}`,
           );
         });
+
+        it('should ignore payment processor fee if host.settings.transferwise.ignorePaymentProcessorFees is true', async () => {
+          await host.update({
+            settings: defaultsDeep(host.settings, { transferwise: { ignorePaymentProcessorFees: true } }),
+          });
+          const mutationParams = { expenseId: expense.id, action: 'PAY' };
+          const res = await graphqlQueryV2(processExpenseMutation, mutationParams, hostAdmin);
+          res.errors && console.error(res.errors);
```

Comment:
Probably not needed, 😉 

```suggestion
```

Reply:
@kewitz : Am I mistaken or does this line have some purpose. Just wondering. 😄

Reply:
@SudharakaP, a GraphQL call won't throw if there are any errors because the call is a bundle of resolvers.
It will return a property containing an array of every resolver that failed and its path.
We often include this line in our tests for debugging purposes, if there's an error we want to print it so we can properly see its stack trace.